tracee-ebpf/tracee/net_events.go

```diff
@@ -63,11 +182,11 @@ func (t *Tracee) processNetEvents() {
 						t.handleError(err)
 						continue
 					}
-					networkProcess, err := t.getProcessCtx(int(hostTid))
+
 					hr, min, sec := timeStampObj.Clock()
 					nsec := uint16(timeStampObj.Nanosecond())
 
-					if err != nil {
+					if networkThreadErr != nil {
```

Comment:
if networkThreadErr != nil you will never reach this point (as you continue in line 138 above)